lib/ProfilePicture/ProfilePicture.js

```diff
@@ -0,0 +1,47 @@
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Img } from 'react-image';
+
+import { Loading } from '@folio/stripes-components';
+
+import { isAValidURL } from './utils/util';
+import useProfilePicture from './utils';
+import profilePicThumbnail from './icons/profilePicThumbnail.png';
+
+import css from './ProfilePicture.css';
+
+const ProfilePicture = ({ profilePictureLink, croppedLocalImage }) => {
+  const intl = useIntl();
+  const { isFetching, profilePictureData } = useProfilePicture({ profilePictureId: profilePictureLink });
```

Comment:
Based on the comments below, it seems `profilePictureLink` (which you refer to as "Profile Picture Link" on line 18, I think?) is not necessarily a link, right? Then don't put `Link` in its name. 

The logic here seems reversed. If we have a link, we don't need the hook to query for the data, do we? Can we check that first, moving the logic up from line 23 before line 15, and then pass a `disabled` prop to the hook based on whether it's a valid URL?

Reply:
Hey Zak, renamed profilePictureLink to profilePictureSource and moved the logic of validating whether whether profilePictureLink is URL or not before the useProfilePicture hook, created a **isProfilePictureAUUID** prop while replaces the isAValidUUID utility function inside useProfilePicture hook.

Reply:
Hey Zak, renamed profilePictureLink to **profilePictureSource** and shifted the logic of validating whether profilePictureLink is URL or not before the useProfilePicture hook.
Created **isProfilePictureAUUID** prop which replaces the isAValidUUID utility function inside useProfilePicture hook.